enhancements/sig-architecture/17-cluster-features/README.md

```diff
@@ -0,0 +1,110 @@
+# Managed cluster feature labels
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+There are requirements about selecting `ManagedClusters` based on the certain features of the managed cluster by `Placement` API. These requirements include:
+
+1. Select a set of the clusters that a certain `ManagedClusterAddon` is enabled. For instance, a user wants to install an application that needs multicluster service connectivity, the use will need to select a set of clusters that `Submariner` addon is enabled by using `Placement` API.
+2. Select a set of the clusters with a certain capability. For instance, a user wants to deploy an AI workload to a set of the clusters in which GPUs are installed.
+
+## Motivation
+
+This proposal is to standardise the labels and `ClusterClaims` that are categorized as features of a `ManagedCluster`.  The idea is inspired by [node features](https://kubernetes-sigs.github.io/node-feature-discovery/) in kubernetes.
+
+### Goals
+
+- define the format of labels and `ClusterClaims` for `ManagedClusters` to represent a feature in managed cluster
+- implement addon as feature labels in `registration`.
+
```

Comment:
If implemented in registration will the label be removed when the addon is disabled?

Reply:
it will be removed when related `ManagedClusterAddon`  resource is deleted. I will add this in the doc.